flink-connector/src/test/java/com/datastrato/gravitino/flink/connector/integration/test/hive/FlinkHiveCatalogIT.java

```diff
@@ -233,6 +238,125 @@ public void testGetCatalogFromGravitino() {
     tableEnv.executeSql("drop catalog " + catalogName);
     Assertions.assertFalse(metalake.catalogExists(catalogName));
     Assertions.assertEquals(
-        1, tableEnv.listCatalogs().length, "The created catalog should be dropped.");
+        2, tableEnv.listCatalogs().length, "The created catalog should be dropped.");
+  }
+
+  @Test
+  public void testCreateSchema() {
+    tableEnv.useCatalog(defaultHiveCatalog);
+    com.datastrato.gravitino.Catalog catalog = metalake.loadCatalog(defaultHiveCatalog);
+    String schema = "test_create_schema";
+    try {
+      TableResult tableResult =
+          tableEnv.executeSql(String.format("CREATE DATABASE IF NOT EXISTS %s", schema));
```

Comment:
If the user wants to specify a `location` property, how should SQL be written?

Reply:
@mchades Flink SQL doesn't support setting the `location`. We can use the property `location` in Gravitino. WDYT?